types/node-dijkstra/index.d.ts

```diff
@@ -0,0 +1,146 @@
+// Type definitions for node-dijkstra 2.5
+// Project: https://github.com/albertorestifo/node-dijkstra
+// Definitions by: Jorge López <https://github.com/nokutu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export default class Graph {
```

Comment:
It's not `export default` https://github.com/albertorestifo/node-dijkstra/blob/master/libs/Graph.js#L293